plugin/core/views.py

```diff
@@ -1019,3 +1023,13 @@ def format_code_actions_for_quick_panel(
         if code_action.get('isPreferred', False):
             selected_index = idx
     return items, selected_index
+
+
+def get_insert_replace_support_html(item: CompletionItem) -> Optional[str]:
+    text_edit = item.get('textEdit')
+    if text_edit and 'insert' in text_edit and 'replace' in text_edit:
```

Comment:
Sorry, don't have time to check the spec now, but is it necessary to check for `textEdit`?

Reply:
yes, there is a type error otherwise :)